packages/manager/modules/iam/src/components/advancedPolicySearch/advancedPolicySearch.component.js

```diff
@@ -4,6 +4,9 @@ import template from './advancedPolicySearch.template.html';
 export default {
   bindings: {
     goBack: '<',
+    identitiesFilter: '<',
+    resourcesFilter: '<',
+    actionsFilter: '<',
```

Comment:
Filter sounds like a filter function
```suggestion
    identitiesCriteria: '<',
    resourcesCriteria: '<',
    actionsCriteria: '<',
```